### PR TITLE
add typescript types

### PR DESCRIPTION
**Pull request by ryantate13**
Fix for #34

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        region?: string;
+        token?: string;
+        debug?: boolean;
+    }
+
+    interface ListKeyResult {
+        keys: string[];
+    }
+
+    interface SecureFileSummaries {
```
Comment (kyeotic) [inline index.d.ts:13]: I don't think this name should not be plural, it describe a single instance.

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    class CerberusClient {
+        constructor(options: CerberusClientOptions);
+        writeSecureData(path: string, data: Record<string, string>): Promise<void>;
+        getSecureData(path: string): Promise<Record<string, string>>;
+        deleteSecureData(path: string): Promise<void>;
+        listPathsForSecureData(path: string): Promise<ListKeyResult>;
+        listFile(path: string): Promise<ListFileResult>;
+        readFile(path: string): Promise<Buffer | string>;
```
Comment (kyeotic) [inline index.d.ts:41]: Does this actually return a non-Buffer, non-Error `string` result?

File: index.d.ts (hunk 1 above)
Comment (ryantate13) [reply index.d.ts]: Tackled in most recent commit

**Comment by kyeotic** [comment]
@fieldju I can't recall, does publish auto-increment the PATCH or does that need to be done before merge?

**Comment by melanahammel** [comment]
@kyeotic The version needs to be manually updated, so I would recommend doing it before the merge

**Pull request merged**